dockyard/api/config.py

```diff
@@ -0,0 +1,24 @@
+# Pecan Application Configurations
+app = {
+    'root': 'dockyard.controllers.root.RootController',
+    'modules': ['dockyard', 'dockyard.api'],
+    'static_root': '%(confdir)s/public',
```

Comment:
As per documentation,

The directory where your static files can be found (relative to the project root). Pecan comes with middleware that can be used to serve static files (like CSS and Javascript files) during development.

i think, for out project static files are not required therefor, these it needs to be considered, whether we should keep it or not.